codegen/java-gen/src/main/scala/kalix/codegen/java/ActionTestKitGenerator.scala

```diff
@@ -141,9 +143,9 @@ object ActionTestKitGenerator {
   }
 
   def generateServices(service: ModelBuilder.ActionService): String = {
-    require(!service.commands.isEmpty, "empty `commands` not allowed")
+    val commands = service.commands.filterNot(_.ignore)
```

Comment:
Isn't that previous validation still valid? If an action has no methods something is not right. Maybe also only ignore-methods because that wouldn't make sense either.

Reply:
I don't know why we had this require in the testkit for Java? It's not in the testkit for Scala and not in the actual action generation.
With this additional filter of ignore methods it can be possible to mark all methods as ignore (even though it doesn't make sense).

Reply:
Ah, it's in the testkit, not in the action generator. I had a feeling we did something like that in the action generator after some user feedback. But then it's fine removing I guess.